Server/JokesOnYou.Web.Api/DTOs/UserLoginDTO.cs

```diff
@@ -8,6 +8,7 @@ namespace JokesOnYou.Web.Api.DTOs
     public class UserLoginDTO
     {
         public string Email { get; set; }
+        public string Username { get; set; } 
```

Comment:
Sounds good!

Reply:
> I think we should remove `Email` and `UserName` and change to `LoginName` ? or just `Name` ?

It should be just UserName because username can be any sort of username (email, name)